server/README.md

```diff
@@ -15,15 +16,22 @@ clientID:  'Github Client ID',
 clientSecret:  'Github Client Secret'
 },
 facebook:  {
-clientID:  'FB Client ID',
-clientSecret:  'FB Client Secret'
-},
+clientID: ,
 VAPID_KEYS  :  {
 Public:  "Vapid Keys",
 Private:  "Vapid Keys"
 }}</code>
 
-3.Write <code>npm install</code> , to install node modules.
-4. Write <code>npm run dev</code>, to start server on PORT 3000.
+Add Vapid keys that are given below.
+

```

Comment:
VAPID Keys need to be generated everytime bythe user, so don't paste your VAPID keys here